src/styles/blocks/star.scss

```diff
@@ -0,0 +1,28 @@
+.star {
+  display: flex;
+  margin-bottom: 24px;
+}
+
+.star__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(../images/star.svg);
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.star__star:nth-child(-n + 4) {
```

Comment:
It should be the same functionality as in the stars task, if the block has modifier `stars--3` you should show 3 active stars, and so on